HTML/html_completions.py

```diff
@@ -156,26 +251,46 @@ def get_completions(self, view, prefix, locations, is_inside_tag):
-            "p", "pre", "span", "samp", "select", "small", "strong", "sub", "sup", "table", "tbody", "td", "textarea", "tfoot", "th", "thead", "title",
-            "tr", "tt", "u", "ul", "var", "article", "aside", "audio", "canvas", "footer", "header", "nav", "section", "video"])
+        normal_tags = ([
+            'abbr', 'acronym', 'address', 'applet', 'area', 'article', 'aside',
+            'audio', 'b', 'base', 'basefont', 'bdi', 'bdo', 'big', 'blockquote',
+            'body', 'button', 'center', 'canvas', 'caption', 'cdata',
+            'cite', 'col', 'colgroup', 'code', 'content', 'data', 'datalist',
+            'dir', 'div', 'dd', 'del', 'details', 'dfn', 'dl', 'dt', 'element',
+            'em', 'embed', 'fieldset', 'figure', 'figcaption', 'font', 'footer',
+            'form', 'frame', 'frameset', 'head', 'header', 'h1', 'h2', 'h3',
+            'h4', 'h5', 'h6', 'i', 'input', 'ins', 'isindex', 'kbd', 'keygen',
+            'li', 'label', 'legend', 'main', 'map', 'mark', 'meter',
+            'nav', 'noframes', 'noscript', 'object', 'ol', 'optgroup',
+            'option', 'output', 'p', 'picture', 'pre', 'q', 'rp',
+            'rt', 'rtc', 'ruby', 's', 'samp', 'section', 'select', 'shadow',
+            'small', 'span', 'strong', 'sub', 'summary', 'sup',
+            'table', 'tbody', 'td', 'template', 'textarea', 'tfoot', 'th',
+            'thead', 'time', 'title', 'tr', 'tt', 'u', 'ul', 'var',
+            'video'
+        ])
 
         for tag in normal_tags:
             default_list.append(make_completion(tag))
             default_list.append(make_completion(tag.upper()))
 
         default_list += ([
-            ("a\tTag", "a href=\"$1\">$0</a>"),
-            ("iframe\tTag", "iframe src=\"$1\">$0</iframe>"),
-            ("link\tTag", "link rel=\"stylesheet\" type=\"text/css\" href=\"$1\">"),
-            ("script\tTag", "script type=\"${1:text/javascript}\">$0</script>"),
-            ("style\tTag", "style type=\"${1:text/css}\">$0</style>"),
-            ("img\tTag", "img src=\"$1\">"),
-            ("param\tTag", "param name=\"$1\" value=\"$2\">")
+            ('a\tTag', 'a href=\"$1\">$0</a>'),
+            ('audio\tTag', 'audio src=\"$1\">$0</audio>'),
+            ('iframe\tTag', 'iframe src=\"$1\">$0</iframe>'),
+            ('img\tTag', 'img src=\"$1\">'),
+            ('link\tTag', 'link rel=\"stylesheet\" type=\"text/css\" href=\"$1\">'),
+            ('param\tTag', 'param name=\"$1\" value=\"$2\">'),
+            ('progress\tTag', 'progress value=\"$1\" max=\"$2\">'),
+            ('script\tTag', 'script type=\"${1:text/javascript}\">$0</script>'),
+            ('source\tTag', 'source src=\"$1\" type=\"$2\">'),
+            ('style\tTag', 'style type=\"${1:text/css}\">$0</style>'),
+            ('track\tTag', 'track kind=\"$1\" src=\"$2\">'),
+            ('video\tTag', 'video src=\"$1\">$0</video>')
```

Comment:
> The $0 anchor would just be for "continue typing here, once all parameters have been filled"

@FichteFoll As far as I observed it, having the last caret position after the complete tag is baked in somewhere. What you want is type `img`, hit <kbd>tab</kbd>, have the caret after `src="`, hit <kbd>tab</kbd> again and have the caret after `<img src="">`, right?

If so, it's already working like that without the need to explicitly add `$0`.